src/CopyrighFixer/CopyrighFixer/configparser.cpp

```diff
@@ -0,0 +1,50 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the lgplv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "configparser.h"
+#include <quasarapp.h>
+#include <iostream>
+
+
+namespace CopyrighFixer {
+ConfigParser::ConfigParser() {
+
+}
+
+const Config ConfigParser::parseOptions(const Config &conf) const {
+
+    Config configOption = conf;
+
+    if (QuasarAppUtils::Params::isEndable("sourceDir")) {
+        configOption.setSourceDir(QuasarAppUtils::Params::getArg("sourceDir"));
+    }
+    else {
+        std::cout << "Warning: Not option sourceDir" << std::endl;
```

Comment:
Use the [QuasarApp::params::log ](https://quasarapp.ddns.net:3031/docs/QuasarApp/QuasarAppLib/latest/classQuasarAppUtils_1_1Params.html#ab20a066955d690172a4aaa82889475f3)method for printing messages.

Reply:
Your application can continue works after the fail parse sourceDir ? 
If No then this is not warning and this is error, You need to print error message and return false.